piper deps aren't packaged yet, and should be USE conditional anyway

https://bugs.gentoo.org/961974

--- a/setup/extensions.json
+++ b/setup/extensions.json
@@ -135,14 +135,6 @@
 		"needs_c++": "17"
     },
     {
-        "name": "piper",
-        "sources": "calibre/utils/tts/piper.cpp",
-		"needs_c++": "17",
-        "libraries": "!piper_libs",
-        "lib_dirs": "!piper_lib_dirs",
-        "inc_dirs": "!piper_inc_dirs"
-    },
-    {
         "name": "html_as_json",
         "sources": "calibre/srv/html_as_json.cpp",
 		"needs_c++": "11"
--- a/src/calibre/constants.py
+++ b/src/calibre/constants.py
@@ -255,7 +255,6 @@
             'rcc_backend',
             'icu',
             'speedup',
-            'piper',
             'html_as_json',
             'fast_css_transform',
             'translator',
--- a/src/calibre/utils/run_tests.py
+++ b/src/calibre/utils/run_tests.py
@@ -567,6 +567,7 @@
             }
         if 'SKIP_SPEECH_TESTS' in os.environ:
             exclude_packages.add('calibre.gui2.tts')
+            exclude_modules.add('calibre.utils.tts.piper')
         if not isbsd:
             exclude_modules.add('calibre.devices.usbms.hal')
         d = os.path.dirname
